mongo/collection.go

```diff
@@ -298,7 +298,7 @@ func (coll *Collection) insert(ctx context.Context, documents []interface{},
 	if imo.Ordered != nil {
 		op = op.Ordered(*imo.Ordered)
 	}
-	retry := driver.RetryNone
+	retry := driver.RetryTimeout
 	if coll.client.retryWrites {
 		retry = driver.RetryOncePerCommand
```

Comment:
It seems like this will enable retrying writes when `timeoutMS` is set, even if `retryWrites == false`. Is that correct? How does a user disable retrying writes when `timeoutMS` is set?